### ISSUE TITLE
Early stopping

### ISSUE THREAD

**Issue by snowood1** [chrisdonahue/ilm#8]
Where did you implement the early stopping based on PPL on the validation set in the training script? Thanks
